source/code/plugins/out_oms_changetracking_file.rb

```diff
@@ -295,7 +307,7 @@ def get_changed_files(records)
                 if !@@ContentlocationUri.nil? and !@@ContentlocationUri.empty? and !collection.empty?
                    key = collection["CollectionName"]
                    date = collection["DateModified"]
-                   fileName = File.basename(key) + date
+                   fileName = date + File.basename(key)
```

Comment:
If possible, "-" or a delim would help it making a little more readable. (If the names are consumed anywhere, like a customer might open up the storage account)